### ISSUE TITLE
Doubt

### ISSUE THREAD

**Issue by yuhkuns2** [Excrulon/Tree-of-Savior-Magnum-Opus-Puzzles#1]
Is it really possible to craft honeymeli/gold bar etc in itos actually ? and can give me a print screen/info how column/row numbers means ? ty

**Comment by Excrulon** [comment]
No, those are brand new recipes in the ktos rank 8 patch. itos will get them with rank 8 as well. Just count the rows/columns starting at 0.

**Comment by yuhkuns2** [comment]
I really didn't understand how columns/rows number works but thanks for the answer, like honeymeli column 0,1,2,1,1 row 0,0,1,2,3 what exactly that numbers means ?
